syncstorage/storage/sql/dbconnect.py

```diff
@@ -144,21 +159,85 @@ def _get_bso_columns(table_name):
 
 bso = Table("bso", metadata, *_get_bso_columns("bso"))
 
+# Table mapping (user_id, collection_id) => batch IDs
+
+batch_uploads = Table(
+    "batch_uploads",
+    metadata,
+    Column("batch", BigInteger, primary_key=True, nullable=False),
```

Comment:
Additional thoughts: changing the type for `batch` means we can use version 4 UUIDs as identifiers directly without dealing with converting to and from integers to store them.  I'm not certain if that would affect the `batch_upload_items` table's purging performance.  I believe it wouldn't but would definitely want to verify experimentally first.